src/components/Timeline/Views/Frames/Frames.tsx

```diff
@@ -170,6 +171,36 @@ export const Frames: FC<TimelineViewProps> = ({
     return value - currentOffsetX + timelineStartOffset;
   }, [position, currentOffsetX, step, length]);
 
+  const onFrameScrub = useCallback((e: MouseEvent) => {
+    const getMouseToFrame = (e: MouseEvent | globalThis.MouseEvent) => {
+      const mouseOffset = e.pageX - scrollable.current!.getBoundingClientRect().left - timelineStartOffset;
```

Comment:
`timelineStartOffset` should this not be in the useCallback deps array?

Reply:
no, it's not intended to change dynamically, so there's no need to update the callback